notebooks/002-openvino-api/002-openvino-api.ipynb

```diff
@@ -2,13 +2,16 @@
  "cells": [
   {
    "cell_type": "markdown",
+   "id": "b875b31c",
```

Comment:
@helena-intel  just curious, but why do these ids show up now? Do we use them for something?

Reply:
This is a change in the notebook format. This is not the best link but gives some info and issues https://github.com/jupyterlab/jupyterlab/issues/9729#issuecomment-803197368